Classes/Public/TextKit/AztecVisualEditor.swift

```diff
@@ -46,6 +45,49 @@ public class AztecVisualEditor : NSObject
     }
 
 
+    // MARK: - Misc helpers
```

Comment:
I'd consider replacing "Misc helpers" with "Paragraph" (or something similar, since it'd be more specific to what's inside the group).